apps/worker/src/app/workflow/services/metric-queue-active.service.ts

```diff
@@ -0,0 +1,115 @@
+import { WorkerOptions } from 'bullmq';
+import { Inject, Injectable, Logger } from '@nestjs/common';
+import { QueueService } from '@novu/application-generic';
+import { checkinForCronJob } from '../../shared/utils/cronHealth';
+import * as process from 'process';
+
+const LOG_CONTEXT = 'MetricQueueService';
+const METRIC_JOB_ID = 'metric-job';
+
+@Injectable()
+export class MetricQueueActiveService extends QueueService<Record<string, never>> {
+  constructor(@Inject('BULLMQ_LIST') private token_list: QueueService[]) {
```

Comment:
Do we have all of the available Queues in this list, from Worker and API?

Reply:
@titenkov Could we get your input on this?
I remember there was a specific reason we did this but I did not write it down or remember.

Reply:
@djabarovgeorge I think that we are missing the one from the `inbound-mail`, its name is `JobTopicNameEnum.INBOUND_PARSE_MAIL`. 
@Cliftonz However, I'm not sure if we should trace it here.

Reply:
@LetItRock we should but I will have to put in a ticket for that as we need to do a few things to get analytics and metrics from the inbound-mail service.